includes/functions/llms.functions.template.php

```diff
@@ -193,23 +193,24 @@ function llms_locate_template( $template_name, $template_path = '', $default_pat
 }
 
 /**
- * Get template override
+ * Get template override.
  *
  * @since Unknown
  * @since 4.8.0 Move template override directories logic into llms_get_template_override_directories.
+ * @since [version] Added `$extension`, and `$is_block_template` parameter.
  *
- * @param string $template Template file.
+ * @param string $template  Template file.
+ * @param string $extension The file extension.
  * @return mixed Template file or false if none exists.
  */
-function llms_get_template_override( $template = '' ) {
+function llms_get_template_override( $template = '', $extension = '' ) {
```

Comment:
This means that block templates and php templates can both be stored in the same override directory, right? 

A new block template (in LifterLMS) is stored in `templates/block-templates/archive-course.html` and a php template could be at `templates/archive-course.php`

So if a theme wanted to override both of these, the directory would have them at `lifterlms/archive-course.html` and `lifterlms/archive-course.php` (both in the same folder)

Right?

I think this is fine but I just want to make sure I'm reading this right.

Reply:
Nope, themes have to override the block templates in
`lifterlms/block-templates/`

The reason for `$extension` is that... I think we just don't need it anymore. I added it when I was doing something else, but it's not needed. I'll remove it. :D